metis-indexing/metis-indexing-client/src/test/java/eu/europeana/indexing/client/test/PublishingServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package eu.europeana.indexing.client.test;
```

Comment:
Test classes are typically in the same package as the classes that they are supposed to test. In fact this class tests something in a completely different project! 

I think the solution here (as mentioned before) is to merge the two projects.

Reply:
Agreed. Fixed.